tests/commands/verify_user.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import pytest
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_verify_user_nouser(capfd):
+    with pytest.raises(CommandError) as e:
+        call_command('verify_user')
+    assert "Either provide a 'user' to verify or use '--all'" in str(e)
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_verify_user_unkonwuser(capfd):
+    with pytest.raises(CommandError) as e:
+        call_command('verify_user', 'not_a_user')
+    assert "User not_a_user does not exist" in str(e)
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_verify_user_noemail(capfd, member):
+    call_command('verify_user', 'member')
+    out, err = capfd.readouterr()
+    assert "You cannot verify an account with no email set" in out
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_verify_user_hasemail(capfd, member_with_email):
+    call_command('verify_user', 'member_with_email')
+    out, err = capfd.readouterr()
+    assert "User 'member_with_email' has been verified" in out
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_verify_user_all(capfd, member_with_email, member2):
+    member2.email = "member2@this.test"
+    member2.save()
+    call_command('verify_user', '--all')
+    out, err = capfd.readouterr()
+    assert "Verified user 'member_with_email'" in out
+    assert "Verified user 'member2'" in out
```

Comment:
Yip probably overkill.  But it does test a slightly different aspect of the command.

I really though don't want to go into such depth with the more complex commands.  For this simple reasons; do we want absolutely full test coverage or do we want to move to Django 1.9, can't have both and I'm sure we actually want the later.  I want the former also but not if its going to take days to make these tests.